src/py/flwr/cli/new/templates/app/code/flwr_tune/strategy.py.tpl

```diff
@@ -0,0 +1,73 @@
+"""$project_name: A Flower / FlowerTune app."""
+
+from logging import INFO, WARN
+from typing import List, Tuple, Union
+
+from flwr.common import FitIns, FitRes, Parameters, log, parameters_to_ndarrays
+from flwr.server.client_manager import ClientManager
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.strategy import FedAvg
+
+
+class FlowerTuneLlm(FedAvg):
+    """Customised FedAvg strategy implementation."""
+    def __init__(self, **kwargs):
+        self.curr_comm_cost = 0.0
```

Comment:
I wonder if this as well as the `test_communication_costs` functionality should be done in a separate class. The reason for this is that imagine someone wants to implement a different strategy, then they'll need to also copy the `test_communication_costs` method and remember to do so! But if this a new class (maybe `"CommunicationsTracker"`) is available, it should be easy for people to integrate  I feel. For sure it will be easier to communicate to people that they have to make use of it. what do you think?

Reply:
Yes good idea to have a separate class!